From 20ed0adb931b6f3dd8a6443311fd33de2509ff77 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 21 Mar 2023 10:58:50 +0530
Subject: [PATCH] [Temporary] Commenting the sepolicy version in Android U

Commenting this for now till version 34 is available.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/manifest.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 35672ac..8737ef2 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -168,7 +168,7 @@
         </interface>
     </hal>
     <!--<kernel target-level="5"/>-->
-    <sepolicy>
+    <!--sepolicy>
         <version>33.0</version>
-    </sepolicy>
+    </sepolicy-->
 </manifest>
-- 
2.17.1

